src/Caliburn.Micro.Platform/XamlPlatformProvider.cs

```diff
@@ -19,25 +19,39 @@ public class XamlPlatformProvider : IPlatformProvider {
         private CoreDispatcher dispatcher;
 #else
         private Dispatcher dispatcher;
+#if !NET45
+        private static TaskCompletionSource<object> taskForce;
```

Comment:
I don't think this needs to be static given there's only once instance of the `XamlPlatformProvider`.